test/fixtures/version-6/can-observable-object/observable-object-static-properties-output.js

```diff
@@ -0,0 +1,29 @@
+class Foo extends ObservableObject {
+  static get props() {
+    return {
+      foo: "number"
+    };
+  }
+
+  static seal = true;
+}
+
+class MyTodo extends ObservableObject {
+  static get props() {
+    return {
+      foo: "number"
+    };
+  }
+
+  static seal = true;
+}
+
+class Foo extends ObservableObject {
+  static get props() {
+    return {
+      foo: "number"
+    };
+  }
+
+  static seal = true;
```

Comment:
Can you confirm that with `DefineMap`, this was sealed by default?

Reply:
Done, observable objects are sealed if are not sealed explicitly in DefineMap.